src/app/models/proptypes.js

```diff
@@ -58,6 +81,25 @@ const baseDefaultPropTypes = {
   },
 };
 
+export const baseTextDefaultPropTypes = (type, text) => ({
```

Comment:
The only use of `baseTextDefaultPropTypes` passes in empty strings for `type` and `text`, so I think you should be able to make this into an object instead of a function.

Reply:
Agree with this